### PR TITLE
Filtering Fields

### PR DESCRIPTION
**Pull request by AmyShackles**
# Description

- Changed graphene.List() to DjangoConnectionFilter(), enabling filtering for every type
- Results from every type are restricted to those belonging to their userId (if logged in)
- Results from every type result in an empty array if not logged in

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Created a user and logged in on frontend to test that there were no breaking changes in the application
Tested all new queries to ensure that they were operational.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: POSserver/POSserver/schema.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    server.schemas.contractor.Query,
+    server.schemas.user.Query,
+    server.schemas.part.Query,
+    server.schemas.client.Query,
+    server.schemas.job.Query,
+    graphene.ObjectType,
+):
+    pass
+
+
```
Comment (rawalpunit) [inline POSserver/POSserver/schema.py]: I like the SuperQuery code...I'd get rid of the orphaned 'pass' on line 16

File: POSserver/server/apps.py [hunk 2]
```diff
@@ -2,4 +2,4 @@
 
 
 class ServerConfig(AppConfig):
-    name = 'server'
+    name = "server"
```
Comment (rawalpunit) [inline POSserver/server/apps.py]: heh..double vs single quotes...so annoying!

File: POSserver/server/migrations/0001_initial.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+            model_name='client',
+            name='contractor',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='server.Contractor'),
+        ),
+        migrations.AddField(
+            model_name='client',
+            name='user',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL),
+        ),
+    ]
```
Comment (rawalpunit) [inline POSserver/server/migrations/0001_initial.py]: I loved this part....the whole operations[] and the migrations in them..make code so clear and easy to understand. Nice job!

File: POSserver/server/schemas/note.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+            "modified_at",
+        ]
+        interfaces = (graphene.relay.Node,)
+
+
+class Query(graphene.ObjectType):
+    # notes = graphene.List(Note_Type)
+    notes = DjangoFilterConnectionField(Note_Type)
+
+    def resolve_notes(self, info, **kwargs):
```
Comment (rawalpunit) [inline POSserver/server/schemas/note.py]: So this DjangoFilterConnectionField is what replaced the Graphene.list()?

File: POSserver/POSserver/schema.py (hunk 1 above)
Comment (AmyShackles) [reply POSserver/POSserver/schema.py]: This pass is not orphaned.  SuperQuery has no methods, so 'pass' is necessary syntactically.

**Pull request merged**